### ISSUE TITLE
Installing as a developer

### ISSUE THREAD

**Issue by fsciortino** [ToFuProject/tofu#608]
@Didou09 I started looking at your latest tutorial (https://github.com/ToFuProject/tofu/issues/603#issuecomment-1009929042), but I am struggling to get the `devel` branch to work/install. It doesn't seem to be the same problem that I run into some time ago, and that @lasofivec helped me with. The new error is shown below.

To reproduce the error: I followed exactly the instructions in the "Installing tofu as a developer" section of the docs:  https://tofuproject.github.io/tofu/installation.html#installing-tofu-as-a-developer. I created a brand new conda environment, and everything else as written in that docs section. I'm working on the lxts cluster at IPP. 

Sorry for this rather annoying/boring issue...

<details>
  <summary>Click to expand!</summary>

  (tofu3new) sciof@lxts00:\~\tofu$ pip install -e .[dev] --user
  Obtaining file:///afs/ipp-garching.mpg.de/home/s/sciof/tofu
    Installing build dependencies ... done
    Checking if build backend supports build_editable ... done
    Getting requirements to build wheel ... done
    Preparing metadata (pyproject.toml) ... done
  Requirement already satisfied: requests in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from tofu===1.5.0-80-g4f0625a0) (2.26.0)
  Requirement already satisfied: numpy in /afs/ipp-garching.mpg.de/home/s/sciof/.conda/envs/tofu3new/lib/python3.9/site-packages (from tofu===1.5.0-80-g4f0625a0) (1.22.0)
  Requirement already satisfied: svg.path in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from tofu===1.5.0-80-g4f0625a0) (4.1)
  Requirement already satisfied: scipy in /afs/ipp-garching.mpg.de/home/s/sciof/.conda/envs/tofu3new/lib/python3.9/site-packages (from tofu===1.5.0-80-g4f0625a0) (1.7.3)
  Requirement already satisfied: cython>=0.26 in /afs/ipp-garching.mpg.de/home/s/sciof/.conda/envs/tofu3new/lib/python3.9/site-packages (from tofu===1.5.0-80-g4f0625a0) (0.29.26)
  Requirement already satisfied: matplotlib in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from tofu===1.5.0-80-g4f0625a0) (3.5.0)
  Requirement already satisfied: sphinx-gallery in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from tofu===1.5.0-80-g4f0625a0) (0.10.1)
  Requirement already satisfied: pytest in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from tofu===1.5.0-80-g4f0625a0) (6.2.5)
  Requirement already satisfied: check-manifest in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from tofu===1.5.0-80-g4f0625a0) (0.47)
  Requirement already satisfied: coverage in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from tofu===1.5.0-80-g4f0625a0) (6.2)
  Requirement already satisfied: sphinx in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from tofu===1.5.0-80-g4f0625a0) (4.3.1)
  Requirement already satisfied: sphinx-bootstrap-theme in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from tofu===1.5.0-80-g4f0625a0) (0.8.0)
  Requirement already satisfied: toml in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from check-manifest->tofu===1.5.0-80-g4f0625a0) (0.10.2)
  Requirement already satisfied: build>=0.1 in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from check-manifest->tofu===1.5.0-80-g4f0625a0) (0.7.0)
  Requirement already satisfied: setuptools in /afs/ipp-garching.mpg.de/home/s/sciof/.conda/envs/tofu3new/lib/python3.9/site-packages (from check-manifest->tofu===1.5.0-80-g4f0625a0) (60.5.0)
  Requirement already satisfied: packaging>=20.0 in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from matplotlib->tofu===1.5.0-80-g4f0625a0) (21.3)
  Requirement already satisfied: python-dateutil>=2.7 in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from matplotlib->tofu===1.5.0-80-g4f0625a0) (2.8.2)
  Requirement already satisfied: fonttools>=4.22.0 in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from matplotlib->tofu===1.5.0-80-g4f0625a0) (4.28.3)
  Requirement already satisfied: cycler>=0.10 in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from matplotlib->tofu===1.5.0-80-g4f0625a0) (0.11.0)
  Requirement already satisfied: kiwisolver>=1.0.1 in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from matplotlib->tofu===1.5.0-80-g4f0625a0) (1.3.2)
  Requirement already satisfied: pillow>=6.2.0 in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from matplotlib->tofu===1.5.0-80-g4f0625a0) (8.4.0)
  Requirement already satisfied: setuptools-scm>=4 in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from matplotlib->tofu===1.5.0-80-g4f0625a0) (6.3.2)
  Requirement already satisfied: pyparsing>=2.2.1 in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from matplotlib->tofu===1.5.0-80-g4f0625a0) (3.0.6)
  Requirement already satisfied: pluggy<2.0,>=0.12 in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from pytest->tofu===1.5.0-80-g4f0625a0) (1.0.0)
  Requirement already satisfied: iniconfig in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from pytest->tofu===1.5.0-80-g4f0625a0) (1.1.1)
  Requirement already satisfied: attrs>=19.2.0 in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from pytest->tofu===1.5.0-80-g4f0625a0) (21.2.0)
  Requirement already satisfied: py>=1.8.2 in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from pytest->tofu===1.5.0-80-g4f0625a0) (1.11.0)
  Requirement already satisfied: certifi>=2017.4.17 in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from requests->tofu===1.5.0-80-g4f0625a0) (2021.10.8)
  Requirement already satisfied: charset-normalizer~=2.0.0 in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from requests->tofu===1.5.0-80-g4f0625a0) (2.0.9)
  Requirement already satisfied: idna<4,>=2.5 in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from requests->tofu===1.5.0-80-g4f0625a0) (3.3)
  Requirement already satisfied: urllib3<1.27,>=1.21.1 in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from requests->tofu===1.5.0-80-g4f0625a0) (1.26.7)
  Requirement already satisfied: Pygments>=2.0 in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from sphinx->tofu===1.5.0-80-g4f0625a0) (2.10.0)
  Requirement already satisfied: sphinxcontrib-qthelp in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from sphinx->tofu===1.5.0-80-g4f0625a0) (1.0.3)
  Requirement already satisfied: alabaster<0.8,>=0.7 in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from sphinx->tofu===1.5.0-80-g4f0625a0) (0.7.12)
  Requirement already satisfied: sphinxcontrib-serializinghtml>=1.1.5 in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from sphinx->tofu===1.5.0-80-g4f0625a0) (1.1.5)
  Requirement already satisfied: imagesize in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from sphinx->tofu===1.5.0-80-g4f0625a0) (1.3.0)
  Requirement already satisfied: docutils<0.18,>=0.14 in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from sphinx->tofu===1.5.0-80-g4f0625a0) (0.17.1)
  Requirement already satisfied: sphinxcontrib-devhelp in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from sphinx->tofu===1.5.0-80-g4f0625a0) (1.0.2)
  Requirement already satisfied: sphinxcontrib-htmlhelp>=2.0.0 in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from sphinx->tofu===1.5.0-80-g4f0625a0) (2.0.0)
  Requirement already satisfied: sphinxcontrib-applehelp in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from sphinx->tofu===1.5.0-80-g4f0625a0) (1.0.2)
  Requirement already satisfied: babel>=1.3 in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from sphinx->tofu===1.5.0-80-g4f0625a0) (2.9.1)
  Requirement already satisfied: Jinja2>=2.3 in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from sphinx->tofu===1.5.0-80-g4f0625a0) (3.0.3)
  Requirement already satisfied: sphinxcontrib-jsmath in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from sphinx->tofu===1.5.0-80-g4f0625a0) (1.0.1)
  Requirement already satisfied: snowballstemmer>=1.1 in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from sphinx->tofu===1.5.0-80-g4f0625a0) (2.2.0)
  Requirement already satisfied: pytz>=2015.7 in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from babel>=1.3->sphinx->tofu===1.5.0-80-g4f0625a0) (2021.3)
  Requirement already satisfied: pep517>=0.9.1 in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from build>=0.1->check-manifest->tofu===1.5.0-80-g4f0625a0) (0.12.0)
  Requirement already satisfied: tomli>=1.0.0 in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from build>=0.1->check-manifest->tofu===1.5.0-80-g4f0625a0) (1.2.2)
  Requirement already satisfied: MarkupSafe>=2.0 in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from Jinja2>=2.3->sphinx->tofu===1.5.0-80-g4f0625a0) (2.0.1)
  Requirement already satisfied: six>=1.5 in /afs/ipp-garching.mpg.de/home/s/sciof/.local/lib/python3.9/site-packages (from python-dateutil>=2.7->matplotlib->tofu===1.5.0-80-g4f0625a0) (1.16.0)
  Installing collected packages: tofu
    Running setup.py develop for tofu
  Successfully installed tofu
  (tofu3new) sciof@lxts00:~/tofu$ pytest tofu/tests
  ====================================== test session starts ======================================
  platform linux -- Python 3.8.8, pytest-6.2.3, py-1.10.0, pluggy-0.13.1
  rootdir: /afs/ipp-garching.mpg.de/home/s/sciof/tofu
  plugins: cov-3.0.0, anyio-2.2.0
  collected 0 items / 15 errors                                                                   
  
  ============================================ ERRORS =============================================
  ___________________ ERROR collecting tofu/tests/tests00_root/test_03_plot.py ____________________
  ImportError while importing test module '/afs/ipp-garching.mpg.de/home/s/sciof/tofu/tofu/tests/tests00_root/test_03_plot.py'.
  Hint: make sure your test modules/packages have valid Python names.
  Traceback:
  tofu/geom/_core.py:30: in <module>
      import tofu.geom._GG as _GG
  E   ModuleNotFoundError: No module named 'tofu.geom._GG'
  
  During handling of the above exception, another exception occurred:
  /afs/.ipp/@sys/os/opt/anaconda/3/2021.05/lib/python3.8/importlib/__init__.py:127: in import_module
      return _bootstrap._gcd_import(name[level:], package, level)
  tofu/__init__.py:62: in <module>
      import tofu.geom as geom
  tofu/geom/__init__.py:12: in <module>
      from tofu.geom._core import PlasmaDomain, Ves, PFC, CoilPF, CoilCS, Config
  tofu/geom/_core.py:36: in <module>
      from . import _GG as _GG
  E   ImportError: cannot import name '_GG' from partially initialized module 'tofu.geom' (most likely due to a circular import) (/afs/ipp-garching.mpg.de/home/s/sciof/tofu/tofu/geom/__init__.py)
  ____________________ ERROR collecting tofu/tests/tests01_geom/test_01_GG.py _____________________
  ImportError while importing test module '/afs/ipp-garching.mpg.de/home/s/sciof/tofu/tofu/tests/tests01_geom/test_01_GG.py'.
  Hint: make sure your test modules/packages have valid Python names.
  Traceback:
  tofu/geom/_core.py:30: in <module>
      import tofu.geom._GG as _GG
  E   ModuleNotFoundError: No module named 'tofu.geom._GG'
  
  During handling of the above exception, another exception occurred:
  /afs/.ipp/@sys/os/opt/anaconda/3/2021.05/lib/python3.8/importlib/__init__.py:127: in import_module
      return _bootstrap._gcd_import(name[level:], package, level)
  tofu/__init__.py:62: in <module>
      import tofu.geom as geom
  tofu/geom/__init__.py:12: in <module>
      from tofu.geom._core import PlasmaDomain, Ves, PFC, CoilPF, CoilCS, Config
  tofu/geom/_core.py:36: in <module>
      from . import _GG as _GG
  E   ImportError: cannot import name '_GG' from partially initialized module 'tofu.geom' (most likely due to a circular import) (/afs/ipp-garching.mpg.de/home/s/sciof/tofu/tofu/geom/__init__.py)
  __________________ ERROR collecting tofu/tests/tests01_geom/test_02_compute.py __________________
  ImportError while importing test module '/afs/ipp-garching.mpg.de/home/s/sciof/tofu/tofu/tests/tests01_geom/test_02_compute.py'.
  Hint: make sure your test modules/packages have valid Python names.
  Traceback:
  tofu/geom/_core.py:30: in <module>
      import tofu.geom._GG as _GG
  E   ModuleNotFoundError: No module named 'tofu.geom._GG'
  
  During handling of the above exception, another exception occurred:
  /afs/.ipp/@sys/os/opt/anaconda/3/2021.05/lib/python3.8/importlib/__init__.py:127: in import_module
      return _bootstrap._gcd_import(name[level:], package, level)
  tofu/__init__.py:62: in <module>
      import tofu.geom as geom
  tofu/geom/__init__.py:12: in <module>
      from tofu.geom._core import PlasmaDomain, Ves, PFC, CoilPF, CoilCS, Config
  tofu/geom/_core.py:36: in <module>
      from . import _GG as _GG
  E   ImportError: cannot import name '_GG' from partially initialized module 'tofu.geom' (most likely due to a circular import) (/afs/ipp-garching.mpg.de/home/s/sciof/tofu/tofu/geom/__init__.py)
  ___________________ ERROR collecting tofu/tests/tests01_geom/test_03_core.py ____________________
  ImportError while importing test module '/afs/ipp-garching.mpg.de/home/s/sciof/tofu/tofu/tests/tests01_geom/test_03_core.py'.
  Hint: make sure your test modules/packages have valid Python names.
  Traceback:
  tofu/geom/_core.py:30: in <module>
      import tofu.geom._GG as _GG
  E   ModuleNotFoundError: No module named 'tofu.geom._GG'
  
  During handling of the above exception, another exception occurred:
  /afs/.ipp/@sys/os/opt/anaconda/3/2021.05/lib/python3.8/importlib/__init__.py:127: in import_module
      return _bootstrap._gcd_import(name[level:], package, level)
  tofu/__init__.py:62: in <module>
      import tofu.geom as geom
  tofu/geom/__init__.py:12: in <module>
      from tofu.geom._core import PlasmaDomain, Ves, PFC, CoilPF, CoilCS, Config
  tofu/geom/_core.py:36: in <module>
      from . import _GG as _GG
  E   ImportError: cannot import name '_GG' from partially initialized module 'tofu.geom' (most likely due to a circular import) (/afs/ipp-garching.mpg.de/home/s/sciof/tofu/tofu/geom/__init__.py)
  ________________ ERROR collecting tofu/tests/tests01_geom/test_04_core_optics.py ________________
  ImportError while importing test module '/afs/ipp-garching.mpg.de/home/s/sciof/tofu/tofu/tests/tests01_geom/test_04_core_optics.py'.
  Hint: make sure your test modules/packages have valid Python names.
  Traceback:
  tofu/geom/_core.py:30: in <module>
      import tofu.geom._GG as _GG
  E   ModuleNotFoundError: No module named 'tofu.geom._GG'
  
  During handling of the above exception, another exception occurred:
  /afs/.ipp/@sys/os/opt/anaconda/3/2021.05/lib/python3.8/importlib/__init__.py:127: in import_module
      return _bootstrap._gcd_import(name[level:], package, level)
  tofu/__init__.py:62: in <module>
      import tofu.geom as geom
  tofu/geom/__init__.py:12: in <module>
      from tofu.geom._core import PlasmaDomain, Ves, PFC, CoilPF, CoilCS, Config
  tofu/geom/_core.py:36: in <module>
      from . import _GG as _GG
  E   ImportError: cannot import name '_GG' from partially initialized module 'tofu.geom' (most likely due to a circular import) (/afs/ipp-garching.mpg.de/home/s/sciof/tofu/tofu/geom/__init__.py)
  _________________ ERROR collecting tofu/tests/tests01_geom/test_04_sampling.py __________________
  ImportError while importing test module '/afs/ipp-garching.mpg.de/home/s/sciof/tofu/tofu/tests/tests01_geom/test_04_sampling.py'.
  Hint: make sure your test modules/packages have valid Python names.
  Traceback:
  tofu/geom/_core.py:30: in <module>
      import tofu.geom._GG as _GG
  E   ModuleNotFoundError: No module named 'tofu.geom._GG'
  
  During handling of the above exception, another exception occurred:
  /afs/.ipp/@sys/os/opt/anaconda/3/2021.05/lib/python3.8/importlib/__init__.py:127: in import_module
      return _bootstrap._gcd_import(name[level:], package, level)
  tofu/__init__.py:62: in <module>
      import tofu.geom as geom
  tofu/geom/__init__.py:12: in <module>
      from tofu.geom._core import PlasmaDomain, Ves, PFC, CoilPF, CoilCS, Config
  tofu/geom/_core.py:36: in <module>
      from . import _GG as _GG
  E   ImportError: cannot import name '_GG' from partially initialized module 'tofu.geom' (most likely due to a circular import) (/afs/ipp-garching.mpg.de/home/s/sciof/tofu/tofu/geom/__init__.py)
  ___________________ ERROR collecting tofu/tests/tests02_data/test_03_core.py ____________________
  ImportError while importing test module '/afs/ipp-garching.mpg.de/home/s/sciof/tofu/tofu/tests/tests02_data/test_03_core.py'.
  Hint: make sure your test modules/packages have valid Python names.
  Traceback:
  tofu/geom/_core.py:30: in <module>
      import tofu.geom._GG as _GG
  E   ModuleNotFoundError: No module named 'tofu.geom._GG'
  
  During handling of the above exception, another exception occurred:
  /afs/.ipp/@sys/os/opt/anaconda/3/2021.05/lib/python3.8/importlib/__init__.py:127: in import_module
      return _bootstrap._gcd_import(name[level:], package, level)
  tofu/__init__.py:62: in <module>
      import tofu.geom as geom
  tofu/geom/__init__.py:12: in <module>
      from tofu.geom._core import PlasmaDomain, Ves, PFC, CoilPF, CoilCS, Config
  tofu/geom/_core.py:36: in <module>
      from . import _GG as _GG
  E   ImportError: cannot import name '_GG' from partially initialized module 'tofu.geom' (most likely due to a circular import) (/afs/ipp-garching.mpg.de/home/s/sciof/tofu/tofu/geom/__init__.py)
  _________________ ERROR collecting tofu/tests/tests02_data/test_04_core_new.py __________________
  ImportError while importing test module '/afs/ipp-garching.mpg.de/home/s/sciof/tofu/tofu/tests/tests02_data/test_04_core_new.py'.
  Hint: make sure your test modules/packages have valid Python names.
  Traceback:
  tofu/geom/_core.py:30: in <module>
      import tofu.geom._GG as _GG
  E   ModuleNotFoundError: No module named 'tofu.geom._GG'
  
  During handling of the above exception, another exception occurred:
  /afs/.ipp/@sys/os/opt/anaconda/3/2021.05/lib/python3.8/importlib/__init__.py:127: in import_module
      return _bootstrap._gcd_import(name[level:], package, level)
  tofu/__init__.py:62: in <module>
      import tofu.geom as geom
  tofu/geom/__init__.py:12: in <module>
      from tofu.geom._core import PlasmaDomain, Ves, PFC, CoilPF, CoilCS, Config
  tofu/geom/_core.py:36: in <module>
      from . import _GG as _GG
  E   ImportError: cannot import name '_GG' from partially initialized module 'tofu.geom' (most likely due to a circular import) (/afs/ipp-garching.mpg.de/home/s/sciof/tofu/tofu/geom/__init__.py)
  _________________ ERROR collecting tofu/tests/tests03_openadas/test_03_core.py __________________
  ImportError while importing test module '/afs/ipp-garching.mpg.de/home/s/sciof/tofu/tofu/tests/tests03_openadas/test_03_core.py'.
  Hint: make sure your test modules/packages have valid Python names.
  Traceback:
  tofu/geom/_core.py:30: in <module>
      import tofu.geom._GG as _GG
  E   ModuleNotFoundError: No module named 'tofu.geom._GG'
  
  During handling of the above exception, another exception occurred:
  /afs/.ipp/@sys/os/opt/anaconda/3/2021.05/lib/python3.8/importlib/__init__.py:127: in import_module
      return _bootstrap._gcd_import(name[level:], package, level)
  tofu/__init__.py:62: in <module>
      import tofu.geom as geom
  tofu/geom/__init__.py:12: in <module>
      from tofu.geom._core import PlasmaDomain, Ves, PFC, CoilPF, CoilCS, Config
  tofu/geom/_core.py:36: in <module>
      from . import _GG as _GG
  E   ImportError: cannot import name '_GG' from partially initialized module 'tofu.geom' (most likely due to a circular import) (/afs/ipp-garching.mpg.de/home/s/sciof/tofu/tofu/geom/__init__.py)
  _________________ ERROR collecting tofu/tests/tests04_spectro/test_01_fit12d.py _________________
  ImportError while importing test module '/afs/ipp-garching.mpg.de/home/s/sciof/tofu/tofu/tests/tests04_spectro/test_01_fit12d.py'.
  Hint: make sure your test modules/packages have valid Python names.
  Traceback:
  tofu/geom/_core.py:30: in <module>
      import tofu.geom._GG as _GG
  E   ModuleNotFoundError: No module named 'tofu.geom._GG'
  
  During handling of the above exception, another exception occurred:
  /afs/.ipp/@sys/os/opt/anaconda/3/2021.05/lib/python3.8/importlib/__init__.py:127: in import_module
      return _bootstrap._gcd_import(name[level:], package, level)
  tofu/__init__.py:62: in <module>
      import tofu.geom as geom
  tofu/geom/__init__.py:12: in <module>
      from tofu.geom._core import PlasmaDomain, Ves, PFC, CoilPF, CoilCS, Config
  tofu/geom/_core.py:36: in <module>
      from . import _GG as _GG
  E   ImportError: cannot import name '_GG' from partially initialized module 'tofu.geom' (most likely due to a circular import) (/afs/ipp-garching.mpg.de/home/s/sciof/tofu/tofu/geom/__init__.py)
  _____________ ERROR collecting tofu/tests/tests04_spectro/test_02_analysistools.py ______________
  ImportError while importing test module '/afs/ipp-garching.mpg.de/home/s/sciof/tofu/tofu/tests/tests04_spectro/test_02_analysistools.py'.
  Hint: make sure your test modules/packages have valid Python names.
  Traceback:
  tofu/geom/_core.py:30: in <module>
      import tofu.geom._GG as _GG
  E   ModuleNotFoundError: No module named 'tofu.geom._GG'
  
  During handling of the above exception, another exception occurred:
  /afs/.ipp/@sys/os/opt/anaconda/3/2021.05/lib/python3.8/importlib/__init__.py:127: in import_module
      return _bootstrap._gcd_import(name[level:], package, level)
  tofu/__init__.py:62: in <module>
      import tofu.geom as geom
  tofu/geom/__init__.py:12: in <module>
      from tofu.geom._core import PlasmaDomain, Ves, PFC, CoilPF, CoilCS, Config
  tofu/geom/_core.py:36: in <module>
      from . import _GG as _GG
  E   ImportError: cannot import name '_GG' from partially initialized module 'tofu.geom' (most likely due to a circular import) (/afs/ipp-garching.mpg.de/home/s/sciof/tofu/tofu/geom/__init__.py)
  ___________________ ERROR collecting tofu/tests/tests05_nist/test_03_core.py ____________________
  ImportError while importing test module '/afs/ipp-garching.mpg.de/home/s/sciof/tofu/tofu/tests/tests05_nist/test_03_core.py'.
  Hint: make sure your test modules/packages have valid Python names.
  Traceback:
  tofu/geom/_core.py:30: in <module>
      import tofu.geom._GG as _GG
  E   ModuleNotFoundError: No module named 'tofu.geom._GG'
  
  During handling of the above exception, another exception occurred:
  /afs/.ipp/@sys/os/opt/anaconda/3/2021.05/lib/python3.8/importlib/__init__.py:127: in import_module
      return _bootstrap._gcd_import(name[level:], package, level)
  tofu/__init__.py:62: in <module>
      import tofu.geom as geom
  tofu/geom/__init__.py:12: in <module>
      from tofu.geom._core import PlasmaDomain, Ves, PFC, CoilPF, CoilCS, Config
  tofu/geom/_core.py:36: in <module>
      from . import _GG as _GG
  E   ImportError: cannot import name '_GG' from partially initialized module 'tofu.geom' (most likely due to a circular import) (/afs/ipp-garching.mpg.de/home/s/sciof/tofu/tofu/geom/__init__.py)
  __________________ ERROR collecting tofu/tests/tests06_mesh/test_01_checks.py ___________________
  ImportError while importing test module '/afs/ipp-garching.mpg.de/home/s/sciof/tofu/tofu/tests/tests06_mesh/test_01_checks.py'.
  Hint: make sure your test modules/packages have valid Python names.
  Traceback:
  tofu/geom/_core.py:30: in <module>
      import tofu.geom._GG as _GG
  E   ModuleNotFoundError: No module named 'tofu.geom._GG'
  
  During handling of the above exception, another exception occurred:
  /afs/.ipp/@sys/os/opt/anaconda/3/2021.05/lib/python3.8/importlib/__init__.py:127: in import_module
      return _bootstrap._gcd_import(name[level:], package, level)
  tofu/__init__.py:62: in <module>
      import tofu.geom as geom
  tofu/geom/__init__.py:12: in <module>
      from tofu.geom._core import PlasmaDomain, Ves, PFC, CoilPF, CoilCS, Config
  tofu/geom/_core.py:36: in <module>
      from . import _GG as _GG
  E   ImportError: cannot import name '_GG' from partially initialized module 'tofu.geom' (most likely due to a circular import) (/afs/ipp-garching.mpg.de/home/s/sciof/tofu/tofu/geom/__init__.py)
  ______________ ERROR collecting tofu/tests/tests07_inversions/test_01_isotropic.py ______________
  ImportError while importing test module '/afs/ipp-garching.mpg.de/home/s/sciof/tofu/tofu/tests/tests07_inversions/test_01_isotropic.py'.
  Hint: make sure your test modules/packages have valid Python names.
  Traceback:
  tofu/geom/_core.py:30: in <module>
      import tofu.geom._GG as _GG
  E   ModuleNotFoundError: No module named 'tofu.geom._GG'
  
  During handling of the above exception, another exception occurred:
  /afs/.ipp/@sys/os/opt/anaconda/3/2021.05/lib/python3.8/importlib/__init__.py:127: in import_module
      return _bootstrap._gcd_import(name[level:], package, level)
  tofu/__init__.py:62: in <module>
      import tofu.geom as geom
  tofu/geom/__init__.py:12: in <module>
      from tofu.geom._core import PlasmaDomain, Ves, PFC, CoilPF, CoilCS, Config
  tofu/geom/_core.py:36: in <module>
      from . import _GG as _GG
  E   ImportError: cannot import name '_GG' from partially initialized module 'tofu.geom' (most likely due to a circular import) (/afs/ipp-garching.mpg.de/home/s/sciof/tofu/tofu/geom/__init__.py)
  ________________ ERROR collecting tofu/tests/tests09_tutorials/test_01_runall.py ________________
  ImportError while importing test module '/afs/ipp-garching.mpg.de/home/s/sciof/tofu/tofu/tests/tests09_tutorials/test_01_runall.py'.
  Hint: make sure your test modules/packages have valid Python names.
  Traceback:
  tofu/geom/_core.py:30: in <module>
      import tofu.geom._GG as _GG
  E   ModuleNotFoundError: No module named 'tofu.geom._GG'
  
  During handling of the above exception, another exception occurred:
  /afs/.ipp/@sys/os/opt/anaconda/3/2021.05/lib/python3.8/importlib/__init__.py:127: in import_module
      return _bootstrap._gcd_import(name[level:], package, level)
  tofu/__init__.py:62: in <module>
      import tofu.geom as geom
  tofu/geom/__init__.py:12: in <module>
      from tofu.geom._core import PlasmaDomain, Ves, PFC, CoilPF, CoilCS, Config
  tofu/geom/_core.py:36: in <module>
      from . import _GG as _GG
  E   ImportError: cannot import name '_GG' from partially initialized module 'tofu.geom' (most likely due to a circular import) (/afs/ipp-garching.mpg.de/home/s/sciof/tofu/tofu/geom/__init__.py)
  ==================================== short test summary info ====================================
  ERROR tofu/tests/tests00_root/test_03_plot.py
  ERROR tofu/tests/tests01_geom/test_01_GG.py
  ERROR tofu/tests/tests01_geom/test_02_compute.py
  ERROR tofu/tests/tests01_geom/test_03_core.py
  ERROR tofu/tests/tests01_geom/test_04_core_optics.py
  ERROR tofu/tests/tests01_geom/test_04_sampling.py
  ERROR tofu/tests/tests02_data/test_03_core.py
  ERROR tofu/tests/tests02_data/test_04_core_new.py
  ERROR tofu/tests/tests03_openadas/test_03_core.py
  ERROR tofu/tests/tests04_spectro/test_01_fit12d.py
  ERROR tofu/tests/tests04_spectro/test_02_analysistools.py
  ERROR tofu/tests/tests05_nist/test_03_core.py
  ERROR tofu/tests/tests06_mesh/test_01_checks.py
  ERROR tofu/tests/tests07_inversions/test_01_isotropic.py
  ERROR tofu/tests/tests09_tutorials/test_01_runall.py
  !!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 15 errors during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!
  ====================================== 15 errors in 8.25s =======================================

</details>

**Comment by lasofivec** [comment]
Hey @fsciortino, looks like the installation went well so I have the feeling that it is not looking in the right place for the installed version of tofu.
Would you mind doing `cd tofu; pytest tests` .
I am also curious about what branch did we tested together last time? The default branch you get when cloning the rep is `devel` so I would have imagined we installed it and tested it last time.

**Comment by flothesof** [comment]
I just checked the developer instructions and was able to install correctly on windows.

@fsciortino, I noticed you used the --user flag for pip in your log.

```
pip install -e .[dev] --user
```
This might cause problems (but I’m no expert). @lasofivec did you install like this last time too?

Cheers,
Florian

**Comment by lasofivec** [comment]
> This might cause problems (but I’m no expert). @lasofivec did you install like this last time too?

I don't think we did but can say for sure. I would say that python is looking for the wrong location for tofu and is using firstly the local directory tofu instead of the installation therefore it doesn't find the compiled library GG. But that's just my hunch

**Comment by flothesof** [comment]
Strange.
Maybe using "python setup.py install" would work? Worth trying...

**Comment by fsciortino** [comment]
thank you both for this help/feedback. I was on a quick zoom call with @Didou09 just a few mins ago and we found that there was a nasty interaction between a version of tofu in my `.local` path (probably left from previous tests with pip) and the `devel` installation that I was doing yesterday/today starting from the cloned directory. 

In other words, I think that I messed up, and it wasn't tofu's fault :/

With this, I learnt that pip and setuptools are pretty bad at understanding multiple sources of installations. In my case, python was trying to fetch some libraries from the pip installation and others from the cloned directory. Just a mess....

Anyway, the solution was to delete everything related to tofu from my `.local` directory and then the cloned `devel` directory worked fine! I think that I'm all set. Sorry for taking your time and many thanks for coming to the help!
